### PR TITLE
Server-side `find-first`, cookie expiry.

### PR DESCRIPTION
**Pull request by z00b**
Two random things lying around locally.

### CODE REVIEW

File: src/cljs/parseapp_cljs/parse.cljs [hunk 1]
```diff
@@ -203,7 +203,7 @@ Ported from js->clj to work around Parse insanity with object detection"
   (count (Query. cls)))
 
 (defn find-first [query]
-  (go (first (<? (find query)))))
+  (find (.limit query 1)))
```
Comment (travis) [inline src/cljs/parseapp_cljs/parse.cljs]: will the channel get a single value or a single value wrapped in a vector? I'd think you'd still need to wrap this find in `(go (first (<?` but would believe `find` has special behavior in this case

**Comment by travis** [comment]
:cake: otherwise

**Comment by travis** [comment]
oh, related to the find-first change, did you run the tests? good habit to get into

**Comment by z00b** [comment]
tests.  who knew?

**Comment by travis** [comment]
:cake:

**Pull request merged**